client/components/chat-list-section.jsx

```diff
@@ -8,9 +8,9 @@ export default class ChatListSection extends React.Component {
     this.state = {
       formIsOpen: false,
       form: {
-        chatName: '',
-        userName: ''
+        chatName: ''
       },
+      user: this.props.user,
```

Comment:
Try to avoid putting `props` in state if you can. If all you are doing is reading `this.props.user`, then use `this.props.user` directly in your other methods, instead of `this.state.user`.